src/dotnet-install.ps1

```diff
@@ -122,6 +125,12 @@ if ($SharedRuntime -and (-not $Runtime)) {
     $Runtime = "dotnet"
 }
 
+#FeedCredential should start with "?", for it to be added to the end of the link.
+#adding "?" at the beginning of the FeedCredential if needed.
+if ((![string]::IsNullOrEmpty($FeedCredential)) -and ($FeedCredential[0] -ne '?')) {
```

Comment:
minor: are there other restrictions for the `FeedCredential` string?  If yes, does it make sense to also enforce ithere?

Reply:
The structure of FeedCredential string is unfortunately something that I have little information about. Thinking about that, behind the aka.ms link could be different storages, and it might be that credential strings for them could vary in format. They definitely has to be passed as parameters within, so that's why we need to add them with "?" to the URI.